gendst/data/positions.go

```diff
@@ -280,7 +280,12 @@ A /*Label*/ : /*Colon*/
 
 	// TypeSpec(1)
 	type (
-		/*Start*/ T2 = /*Name*/ T1 /*End*/
+		/*Start*/ T2 = /*AliasName*/ T1 /*End*/
+	)
+
+	// TypeSpec(2)
+	type (
+		X /*Name*/ [P any] /*TypeParams*/ []P
```

Comment:
I think we're missing the `Start` and `End` decorations...

Reply:
Good :eye: !